src/components/message.vue

```diff
@@ -0,0 +1,588 @@
+<template>
+    <div class="comments_item">
+      <div class="comment_user_info">
+        <img
+          :src="
+            require(`../photos/photosForProfile/${commentAuthor.pathToPhoto}`)
+          "
+          :alt="commentAuthor.accountName"
+          width="32"
+          height="32"
+        />
+        <p class="comment_user_name">{{ commentAuthor.accountName }}</p>
+        <span v-if="isCurrentUserComment" class="its_you">you</span>
+        <p class="comment_user_date">{{ dateDifference(comment.date) }}</p>
+      </div>
+  
+      <textarea
+        v-if="isEditable"
+        v-model="textareaValue"
+        class="comment_user_text comment_user_edit_textarea"
+      />
+      <ReferenceChecking v-else :commentText="textareaValue" />
+  
+      <div class="quantity_of_likes_container">
+        <button
+          type="button"
+          class="quantity_calc_button"
+          @click="
+            changeQuantityLikes({
+              id: comment.id,
+              newQuantityOflikes: comment.quantityOflikes + 1,
+            })
+          "
+        >
+          <img width=" 10" height="10" src="../photos/+.svg" alt="minus" />
+        </button>
+        <span class="quantity_of_likes_textholder">{{
+          comment.quantityOflikes
+        }}</span>
+        <button
+          type="button"
+          class="quantity_calc_button"
+          @click="
+            changeQuantityLikes({
+              id: comment.id,
+              newQuantityOflikes: comment.quantityOflikes - 1,
+            })
+          "
+        >
+          <img width="10" height="10" src="../photos/-.svg" alt="minus" />
+        </button>
+      </div>
+      <div v-if="isCurrentUserComment" class="delete_edit_buttons_container">
+        <button
+          @click="setShowModal(true)"
+          :disabled="isEditable"
+          type="button"
+          class="edit_comment_state_button delete_btn"
+        >
+          <Delete /> Delete
+        </button>
+        <button
+          type="button"
+          :disabled="isEditable"
+          @click="setIsEdit(true)"
+          class="edit_comment_state_button edit_btn"
+        >
+          <Edit /> Edit
+        </button>
+      </div>
+      <button
+        v-else
+        class="delete_edit_buttons_container reply_btn"
+        @click="setIsReplying(true)"
+        type="button"
+      >
+        <img src="../photos/Reply.svg" alt="reply" width="14" height="13" /><span
+          class="reply_text"
+          >Reply</span
+        >
+      </button>
+      <button
+        v-if="isEditable"
+        class="udate_comment_text_btn"
+        @click="
+          setIsEdit(false),
+            changeCommentText({
+              id: comment.id,
+              newCommentText: textareaValue,
+            })
+        "
+      >
```

Comment:
if you invoke more then 2 functions in the event handler then create a separate method
```suggestion
        @click="onUpdateClick"
      >
```